soc/renesas/smartbond/da1469x/soc.c

```diff
@@ -146,8 +146,12 @@ static int renesas_da1469x_init(void)
 				CRG_TOP_PMU_CTRL_REG_COM_SLEEP_Msk     |
 				CRG_TOP_PMU_CTRL_REG_RADIO_SLEEP_Msk);
 
-	/* PDC should take care of PD_SYS */
-	CRG_TOP->PMU_CTRL_REG &= ~CRG_TOP_PMU_CTRL_REG_SYS_SLEEP_Msk;
+	/*
+	 * Take PD_SYS control.
+	 * It is released during PRE_KERNEL_2 where a PDC entry is added
+	 * exists for sure.
+	 */
+	da1469x_pd_acquire(MCU_PD_DOMAIN_SYS);
```

Comment:
That API should be called after `da1469x_pd_init()` and `da1469x_trimv_init_from_otp()` have been called. Otherwise, the trim values applied will be invalid.

Reply:
You right. I missed that we apply trim values when we acquire PD_SYS.